Content.Server/Voting/VoteCommands.cs

```diff
@@ -92,6 +108,40 @@ public void Execute(IConsoleShell shell, string argStr, string[] args)
                 options.Options.Add((args[i], i));
             }
 
+            // Set up the webhook payload
+            var _gameTicker = _entitySystem.GetEntitySystem<GameTicker>();
+
+            var payload = new WebhookPayload()
+            {
+                AvatarUrl = _avatarUrl,
+                Username = _serverName,
+                Embeds = new List<Embed>
+                {
+                    new()
+                    {
+                        Title = $"{shell.Player}",
+                        Color = 13438992, // 2353993
+                        Description = options.Title,
+                        Footer = new EmbedFooter
+                        {
+                            Text = $"{_serverName} {_gameTicker.RoundId} {_gameTicker.RunLevel}",
+                        },
+
+                        Fields = new List<Field> {},
```

Comment:
Probably better to delete blank elements? I guess this will work without it

Reply:
This needs to exist and is filled later.